### PR TITLE
Changed the length from float to string

### PR DESCRIPTION
**Pull request by Marauderer97**
Changed the length field's data type in model and api to string from float. @niranjan94 Hope this is what you wanted with reference to #2644

### CODE REVIEW

**Comment by niranjan94** [comment]
@Marauderer97 you're missing the migration files for the model change

**Comment by Marauderer97** [comment]
@niranjan94 I updated the branch

**Comment by Marauderer97** [comment]
@niranjan94 I updated the branch

File: Vagrantfile [hunk 1]
```diff
@@ -71,5 +71,7 @@ Vagrant.configure(2) do |config|
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
   config.vm.provision :shell, :path => "scripts/install.sh"
-
+  config.proxy.http     = "http://proxy.iiit.ac.in:8080"
```
Comment (niranjan94) [inline Vagrantfile]: Please remove these lines since they're specific to your network

**Comment by Marauderer97** [comment]
@niranjan94 Yes, I had added them by mistake, I reverted my change in my final commit

File: Vagrantfile (hunk 1 above)
Comment (Marauderer97) [inline Vagrantfile]: Already fixed in the next commit. Sorry about that.

**Pull request merged**